CLI/controllers/sendSchemaController.go

```diff
@@ -20,15 +20,11 @@ func serialiseVector(attributes map[string]interface{}, attributeName string) {
 	}
 }
 
-// Serialising size & posXY is inefficient but
-// the team wants it for now
-// "size":"[25,29.4,0]" -> "size": "{\"x\":25,\"y\":29.4,\"z\":0}"
 func serialiseStringVector(attr map[string]interface{}, want string) string {
 	var newSize string
 	if size, ok := attr[want].(string); ok {
 		left := strings.Index(size, "[")
 		right := strings.Index(size, "]")
-		coords := []string{"x", "y", "z"}
 
 		if left != -1 && right != -1 {
```

Comment:
Can't we simplify this now?

Reply:
I've rewritten the `serialiseStringVector` and `serialiseFloatVector` so they are a bit simpler and easier to understand now. Their functioning is still the same: extracting some attributes information (such as height, diameter etc.) from the vectors, and returning a vector (in form of a string) with only the necessary elements.